package.json

```
@@ -4,10 +4,14 @@
   "description": "Official Top.gg Node SDK",
   "main": "./dist/index.js",
   "scripts": {
-    "build": "npm ci && tsc",
+    "build": "tsc",
+    "build:ci": "npm ci && tsc",
```

```suggestion
    "build:ci": "npm i --also=dev && tsc",
```
since you gitignore'd the package-lock.json, clean installs don't work.

---

I removed it based on the advice from https://dev.to/gajus/stop-using-package-lock-json-or-yarn-lock-3ddi but I'm reading into this now and committing package-lock.json is about as controversial as using semicolons lol